### DISCUSSION TITLE
How to write query for fetching foreign key fields - many to many relation ?

### DISCUSSION THREAD

**Discussion by ivishalgandhi** [pocketbase/pocketbase#4852] [Q&A] [answered]
How to write query for fetching foreign key fields - many to many relation ? 

team_member.name ~ '%Abc%'

only records with Abc are returned. If row has two names Abc and Def there are no records returned. Please advise.

**Comment by ganigeorgiev** [comment, marked as answer]
You can try prefixing the operator with `?` for "any/at-least-one" type of constraint:

```js
team_member.name ?~ '%Abc%'
```

(_all available operators can be found in https://pocketbase.io/docs/api-rules-and-filters/#filters-syntax_)

**Comment by ivishalgandhi** [comment]
Worked 👍  Thank you.
